### PR TITLE
Rent model

### PR DESCRIPTION
**Pull request by mcolombo87**
### Summary
New Rent Model with references to User and Books.
### ScreenShoots
image
image
### Trello
https://trello.com/c/KDxD8S1h/20-crear-modelo-de-rent

### CODE REVIEW

File: app/models/rent.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class Rent < ApplicationRecord
+  belongs_to :user
```
Comment (facucompe) [inline app/models/rent.rb:2]: What about users? The rent has an user, and the user `has_many :rents`

Comment (facucompe) [reply app/models/rent.rb:2]: Also has_many requires to add `dependent` statement

**Review by facucompe — CHANGES_REQUESTED** [review]
There are changes from previous PRs, check conflicts

**Review by facucompe — APPROVED** [review]

File: app/models/rent.rb (hunk 1 above)
Comment (mcolombo87) [reply app/models/rent.rb:2]: Done!.

**Pull request merged**